test/e2e/common/container_probe.go

```diff
@@ -123,6 +123,11 @@ var _ = framework.KubeDescribe("Probing container", func() {
 		}, 1, defaultObservationTimeout)
 	})
 
+	/*
+	  Testname: pods-exec-liveness-probe-not-restarted
```

Comment:
Is there any build jiggery to check and enforce here?

Reply:
the PR will just look for well-known tags (like `Testname:`) and do special stuff on those, but otherwise its just a pass through.  We can certainly add more logic to do things like "fail if its not in a format we like".  We should document those requests on the PR I referenced so it can be modified or done in a follow-on PR.

Reply:
I think folks are relying on the results, we can gate the build.

Reply:
Tracking the request here: https://github.com/kubernetes/kubernetes/issues/53810